sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1817,13 +1817,13 @@ public Bound withWriteDisposition(WriteDisposition writeDisposition) {
        */
       public Bound withoutValidation() {
         return new Bound(name, jsonTableRef, tableRefFunction, jsonSchema, createDisposition,
-            writeDisposition, false, bigQueryServices);
+           writeDisposition, false, bigQueryServices);
```

Comment:
a lot of spacing changes in this file, not consistently applied. 2 spaces, or 3, or 4, or 8? (8 at L1810, 3 here, 2 at L1826).